pkg/daemon/ceph/client/filesystem.go

```diff
@@ -174,6 +177,23 @@ func IsMultiFSEnabled() bool {
 	}
 	return false
 }
+func SetPoolSize(context *clusterd.Context, clusterName, fsName string, dataPool []*model.Pool, metdataPool *model.Pool) error {
+	// updating the dataPool size
+	metaDataPoolname := fsName + metaDataPoolSuffix
+	err := SetPoolProperty(context, clusterName, metaDataPoolname, "size", strconv.Itoa(int(metdataPool.ReplicatedConfig.Size)))
```

Comment:
How about making the property name as constants?

Reply:
can be done in a separate PR.

Reply:
Because of the `fsName` is declared within the func scope and that's why I'm concatenating it with the other const that I defined above in this scope.

Reply:
I mean `size` which is getting passed to SetPoolProperty.

Reply:
and there are two functions called one after the another so I think instead of "resetting" the strings, we can bear the cost of one more variable?.